ocean_lib/models/test/test_order.py

```diff
@@ -0,0 +1,81 @@
+#
+# Copyright 2021 Ocean Protocol Foundation
+# SPDX-License-Identifier: Apache-2.0
+#
+import os
+
+from eth_utils import remove_0x_prefix
+
+from ocean_lib.assets.asset import Asset
+from ocean_lib.common.agreements.service_agreement import ServiceAgreement
+from ocean_lib.common.agreements.service_types import ServiceTypes
+from ocean_lib.models.data_token import DataToken
+from ocean_lib.models.order import Order
+from ocean_lib.ocean.util import from_base_18
+from tests.resources.ddo_helpers import get_registered_ddo, get_metadata
+from tests.resources.helper_functions import mint_tokens_and_wait
+
+
+def test_order(alice_ocean, alice_wallet):
+    asset = get_registered_ddo(alice_ocean, get_metadata(), alice_wallet)
+    assert isinstance(asset, Asset)
+    assert asset.data_token_address
+
+    dt = DataToken(asset.data_token_address)
+    mint_tokens_and_wait(dt, alice_wallet.address, alice_wallet)
+
+    service = asset.get_service(service_type=ServiceTypes.ASSET_ACCESS)
+    sa = ServiceAgreement.from_json(service.as_dictionary())
+
+    order_requirements = alice_ocean.assets.order(
+        asset.did, alice_wallet.address, sa.index
+    )
+    assert order_requirements
+
+    args = [
+        order_requirements.amount,
+        order_requirements.data_token_address,
+        asset.did,
+        service.index,
+        "0xF9f2DB837b3db03Be72252fAeD2f6E0b73E428b9",
```

Comment:
What is this, exactly? If it's one of the test keys, please use that instead of hardcoding it.

Reply:
It's not one of the test private keys, I retrieve it from the `test_market_flow`

Reply:
can you please investigate it? Why is it this address specifically? If it's a random address serving for test purposes, let's define it and reuse it between test_market_flow and test_order.

Reply:
Never mind, just tested with `alice_wallet.address` and it works fine. Mystery resolved.